### PR TITLE
7.2 Search | LRDOCS-6601 | Elasticsearch Configuration

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6601, conversion from 7.1 content. cc @lipusz

### CODE REVIEW

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown [hunk 1]
```diff
@@ -75,22 +76,31 @@ Elasticsearch in remote mode, but @product@ can't connect to Elasticsearch, chec
...
-   >If, for instance, you initially connect to a master node, after sniffing, no further requests will go to that master node, but rather to any data nodes instead. The reason the transport client excludes non-data nodes is to avoid sending search traffic to master only nodes.
-   
-To disable cluster sniffing, add `client.transport.sniff: false` to the `additionalConfigurations` property in Liferay's Elasticsearch connector configuration.
+: Elasticsearch clusters can have multiple node 
+[types](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-node.html#modules-node).
+[Cluster sniffing](https://www.elastic.co/guide/en/elasticsearch/client/java-api/6.5/transport-client.html), 
+enabled by default enabled by default in the @product@ connector, looks for
```
Comment (lipusz) [inline deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: duplicated words

Comment (rbohl) [reply deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: No, it's just an echo in my text editor :smile:

Comment (rbohl) [reply deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: Fixed - thanks!

Comment (lipusz) [reply deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: "of course" it's just your editor :stuck_out_tongue_closed_eyes:

**Comment by sez11a** [comment]
Just started reviewing :)

**Pull request merged**